src/components/AcceptProposalForm.tsx

```diff
@@ -74,7 +78,11 @@ export default function AcceptProposalForm({
           </div>
         </div>
 
-        <Button className={{ root: 'mt-2' }} type="submit">
+        <Button
+          className={{ root: 'mt-2' }}
+          type="submit"
+          disabled={feedbackGiven}
```

Comment:
show a message and hide the entire form instead of just disabling the buttons -> users will be confused why they filled the form completely but cannot submit